### PR TITLE
Add hits to benchmark

### PR DESCRIPTION
**Pull request by lutengda**
# Which issue does this PR close?

Related #.

# Rationale for this change

# What changes are included in this PR?

# Are there any user-facing changes?

### CODE REVIEW

**Review by Benxiaohai001 — APPROVED** [review]

File: benchmark/shell_env.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ export BENCHMARK_DATASET=${BENCHMARK_DATASET:-hits}
 export BENCHMARK_TYPE=${BENCHMARK_TYPE:-"pr"}
 
 # hits
-export HITS_DATA_URL=${HITS_DATA_URL:-"http://127.0.0.1:8902/data.gz"}
+export HITS_DATA_URL=${HITS_DATA_URL:-"/data/hits.parquet"}
```
Comment (yukkit) [inline benchmark/shell_env.sh:15]: Is this variable still useful?

File: benchmark/hits/sql/queries.sql [hunk 2]
```diff
@@ -30,7 +30,7 @@ SELECT REGEXP_REPLACE("Referer", '^https?://(?:www\.)?([^/]+)/.*$', '\1') AS k,
...
 SELECT "SearchEngineID", "ClientIP", COUNT(*) AS c, SUM("IsRefresh"), AVG("ResolutionWidth") FROM hits WHERE "SearchPhrase" <> '' GROUP BY "SearchEngineID", "ClientIP" ORDER BY c DESC LIMIT 10;
 SELECT "WatchID", "ClientIP", COUNT(*) AS c, SUM("IsRefresh"), AVG("ResolutionWidth") FROM hits WHERE "SearchPhrase" <> '' GROUP BY "WatchID", "ClientIP" ORDER BY c DESC LIMIT 10;
-SELECT "WatchID", "ClientIP", COUNT(*) AS c, SUM("IsRefresh"), AVG("ResolutionWidth") FROM hits GROUP BY "WatchID", "ClientIP" ORDER BY c DESC LIMIT 10;
+--SELECT "WatchID", "ClientIP", COUNT(*) AS c, SUM("IsRefresh"), AVG("ResolutionWidth") FROM hits GROUP BY "WatchID", "ClientIP" ORDER BY c DESC LIMIT 10;
```
Comment (yukkit) [inline benchmark/hits/sql/queries.sql:33]: You can relate to the issue

File: benchmark/hits/prepare_data.sh [hunk 3]
```diff
@@ -5,17 +5,17 @@ set -e
...
+#    wget --continue ${HITS_DATA_URL} -O /tmp/hits.parquet
+#else
+#    cp ${HITS_DATA_URL} /tmp/hits.parquet
+#fi
 ${SQL_CLI} --file ${current_dir}/sql/create.sql >/dev/null
 q_end=$(date +%s.%N)
 q_time=$(echo "$q_end - $q_start" | bc -l)
 
-data_size=$(wc -c </tmp/hits.parquet)
+data_size=$(wc -c </data/hits.parquet)
```
Comment (yukkit) [inline benchmark/hits/prepare_data.sh:19]: Please remind in README.md ` the hits data must be named hits.parquet under /data` and remove env `HITS_DATA_URL` if necessary

File: benchmark/hits/sql/queries.sql (hunk 2 above)
Comment (lutengda) [reply benchmark/hits/sql/queries.sql:33]: reopen and comment this issue
https://github.com/cnosdb/cnosdb/issues/946

File: benchmark/shell_env.sh (hunk 1 above)
Comment (lutengda) [reply benchmark/shell_env.sh:15]: Yes, I put the data in this location on the gitlab node

File: benchmark/hits/prepare_data.sh (hunk 3 above)
Comment (lutengda) [reply benchmark/hits/prepare_data.sh:19]: ok

File: benchmark/shell_env.sh (hunk 1 above)
Comment (yukkit) [reply benchmark/shell_env.sh:15]: I don't see where this variable is used, could you show me

Comment (lutengda) [reply benchmark/shell_env.sh:15]: I indeed haven't used it, I'll delete it.

**Pull request closed without merging**